src/main/java/org/terasology/core/world/generator/facetProviders/PositionFilters.java

```diff
@@ -67,20 +69,67 @@ private PositionFilters() {
      * Filters based on the density
      *
      * @param density the density facet that contains all tested coords.
-     * @return a predicate that returns true if (density &ge; 0) and (density &lt; 0) for the block at (y - 1)
+     * @return a predicate that returns true if (density <= 0) and (density > 0) for the block at (y - 1)
```

Comment:
This reads funny - I think there is a "for the block at position y" missing somewhere :wink: 
```suggestion
     * @return a predicate that returns true if (density <= 0) at position y and (density > 0) at position (y - 1)
```